qiskit/algorithms/gradients/finite_diff_estimator_gradient.py

```diff
@@ -74,12 +100,25 @@ def _run(
             metadata_.append({"parameters": [circuit.parameters[idx] for idx in indices]})
 
             offset = np.identity(circuit.num_parameters)[indices, :]
-            plus = parameter_values_ + self._epsilon * offset
-            minus = parameter_values_ - self._epsilon * offset
-            n = 2 * len(indices)
-            job = self._estimator.run(
-                [circuit] * n, [observable] * n, plus.tolist() + minus.tolist(), **options
-            )
+            if self._method == "central":
+                plus = parameter_values_ + self._epsilon * offset / 2
```

Comment:
I think this should remain as it was: perturb by `epsilon` in both directions and divide by `2 * epsilon` to make sure we don't change what happens. Especially for finite difference on noisy functions, dividing the distance by 2 could make a difference 🤔

Reply:
This is an API breaking. I believe it is a bug fix because it goes against the general definition.

Reply:
Hmm I'm not sure if this can be considered a bug, in literature you find both versions with eps and eps/2. Personally I think I'm actually more used to see eps and not eps/2, and also SPSA in Qiskit uses an eps shift 🙂

Maybe it would be better to keep it as is, if there is no clear evidence that it's wrong? 😄

Reply:
Well, OK. We just need to adjust epsilon, so personally I don't have the strong opinion, so I'd put it back.

Reply:
I thought it might be better to use `eps` for the denominators of all cases. But, it seems some documents use + an `-eps` for the central method instead of `eps/2`. I think it's ok to leave it as is.